js_modules/dagster-ui/packages/ui-core/src/assets/AssetPageHeader.oss.tsx

```diff
@@ -115,14 +117,17 @@ const TruncatedHeading = styled(Heading)`
   overflow: hidden;
 `;
 
-export const AssetGlobalLineageLink = () => (
-  <Link to="/asset-groups">
-    <Box flex={{gap: 4}}>
-      <Icon color={Colors.linkDefault()} name="lineage" />
-      View lineage
-    </Box>
-  </Link>
-);
+export const AssetGlobalLineageLink = () => {
+  const [assetSelection] = useAssetSelectionState();
+  return (
+    <Link to={globalAssetGraphPathToString({opsQuery: assetSelection, opNames: []})}>
```

Comment:
nice, I think the view preserving logic I have should make sure the view is part of the URL even though this links to the "global" view.

Reply:
Yeah that part seems to work nicely!